service/routes/api.js

```diff
@@ -71,13 +74,18 @@ router.get(/^\/v2\/polyfill(\.\w+)(\.\w+)?/, (req, res) => {
 	res.set('Content-Type', contentTypes[fileExtension]+';charset=utf-8');
 	res.set('Access-Control-Allow-Origin', '*');
 
-	const outputStream = polyfillio.getPolyfillString(params);
-	outputStream.pipe(res, {end: false});
-	outputStream.on('end', () => {
-		if (req.query.callback && req.query.callback.match(/^[\w\.]+$/)) {
-			res.write("\ntypeof "+req.query.callback+"==='function' && "+req.query.callback+"();");
+	const outputStream = mergeStream();
+
+	outputStream.add(polyfillio.getPolyfillString(params));
+
+	if (req.query.callback && req.query.callback.match(/^[\w\.]+$/)) {
+		outputStream.add(streamFromString("\ntypeof "+req.query.callback+"==='function' && "+req.query.callback+"();"));
+	}
```

Comment:
I much prefer this solution to the one in the other PR - the source order still matches the order of output which makes it easier to grok.